bin/image

```diff
@@ -28,6 +28,16 @@ last4minors() {
     done
 }
 
+# filters kube versions which are already pushed
+# missing_from_repo <VERSION...>
+missing_from_repo() {
+    present_tags=$(curl -sSL https://registry.hub.docker.com/v1/repositories/claranet/gcloud-kubectl-docker/tags)
+
+    for version in $@; do
+        echo $present_tags | jq '.[] | .name' | grep $version > /dev/null || echo $version
```

Comment:
please comment what this is about? what are you expecting from the first part?

Reply:
which part?